OracleTuxedo/core/samples/shm/Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Pull Tuxedo base image
-FROM oracle/tuxedo:<VERSION>
+FROM oracle/tuxedo:12.2.2
 
 MAINTAINER Todd Little <todd.little@oracle.com>
 COPY simpapp_runme.sh /home/oracle/
 
 USER root
-RUN chown oracle:oracle -R /home/oracle
+RUN chown oracle:oracle -R /home/oracle/
 
 USER oracle
 WORKDIR /home/oracle
+ENV PATH=.:$PATH
```

Comment:
Avoid magic aliases, like `.` to reference folders (but environment variables are fine of course). 

Please be as explicit and verbose as possible so whoever reads this Dockerfile, knows exactly what is being done.